nexus/src/external_api/http_entrypoints.rs

```diff
@@ -220,13 +224,26 @@ pub fn external_api() -> NexusApiDescription {
         api.register(vpc_subnet_update)?;
         api.register(vpc_subnet_list_network_interfaces)?;
 
+        api.register(vpc_subnet_list_v1)?;
+        api.register(vpc_subnet_view_v1)?;
+        api.register(vpc_subnet_create_v1)?;
+        api.register(vpc_subnet_delete_v1)?;
+        api.register(vpc_subnet_update_v1)?;
+        // api.register(vpc_subnet_list_network_interfaces_v1)?;
```

Comment:
Is this dead code? It looks like there's no such endpoint function itself, so I'd expect we'd just delete this.

Reply:
Maybe add a TODO item to the parent issue? I can't tell if it's already covered, so ignore me if so :)